test/testcafe/spec/Identify_spec.js

```diff
@@ -88,7 +88,7 @@ test.requestHooks(identifyMock)(`should have correct display texts`, async t =>
   await t.expect(identityPageTitle).eql('Sign In');
 
   const rememberMeText = identityPage.getRememberMeText();
-  await t.expect(rememberMeText).eql('Remember Me');
+  await t.expect(rememberMeText).eql('Remember me');
```

Comment:
Have u checked this with UX ?

Reply:
I think it's probably fine given it's same as v1.